src/Common/tests/System/Net/Capability.Security.cs

```diff
@@ -37,6 +37,16 @@ public static bool AreHostsFileNamesInstalled()
             return !(Configuration.Security.HostsFileNamesInstalled == null);
         }
 
+        public static bool ForceUnencryptedHttp2Loopback()
```

Comment:
Why is this called "ForceUnencryptedHttp2Loopback"?  Is this only used when using the HTTP/2 loopback server?

If so, the environment variable should be renamed as well to reflect that.

Otherwise, perhaps this new Capability.* method should be named "ForceUnencryptedHttp2" and be able to be used for any HTTP/2 connection.

Reply:
yes, this is only for HTTP/2 loopback server where we have control over both sides. Tests to external servers will still use HTTPS.

Reply:
Then the environment variable needs a better name I think.

Reply:
what do you suggest @davidsh? 
Http2ForceUnencryptedLoopback and COREFX_HTTP2_FORCEUNENCRYPTEDLOOPBACK ?